### PR TITLE
Possible fix for the double menu

### PR DESCRIPTION
**Pull request by DiegoAndresCortes**
I came up with this idea to try to solve #4882 

Let me know what you think

### CODE REVIEW

**Comment by albertlast** [comment]
From technical side you got here the same issue like the other pr,
the end of line got changed.

File: Themes/default/index.template.php [hunk 1]
```diff
@@ -339,23 +339,21 @@ function template_body_above()
...
+	// Show the menu here, according to the menu sub template, followed by the navigation tree.
 	// Load mobile menu here
 	echo '
 				<a class="menu_icon mobile_user_menu"></a>
-				<div id="mobile_user_menu" class="popup_container">
-					<div class="popup_window description">
-						<div class="popup_heading">', $txt['mobile_user_menu'], '
-						<a href="javascript:void(0);" class="generic_icons hide_popup"></a></div>
-						', template_menu(), '
+				<div id="mobile_user_menu" class="popup_container menu_popup_container">
```
Comment (Gwenwyfar) [inline Themes/default/index.template.php]: Can you use only one main class in the container here? Cleaner html this way (without the menu_popup_heading, etc)

**Review by Gwenwyfar — CHANGES_REQUESTED** [review]
There's also a leftover padding from .popup_window in desktop view, needs to be removed.

**Comment by Gwenwyfar** [comment]
Thanks for doing this. Only fixes the main menu though, could you do the submenus also?

**Comment by DiegoAndresCortes** [comment]
I fixed it I think it's fine now.
I didn't know the genericmenu uses the double menu as well, I'll have a look at it later today

**Comment by jdarwood007** [comment]
Can you fix the line endings please?

**Comment by DiegoAndresCortes** [comment]
Sure, once I finish the genericmenu.

**Comment by DiegoAndresCortes** [comment]
Fixed and updated for genericmenu

**Comment by DiegoAndresCortes** [comment]
By the way, I forgot to mention that I decided to put the html code that wraps the function outside just in case a themer wants to easily replace/remove it, which is very likely.

**Review by Gwenwyfar — APPROVED** [review]

**Pull request merged**